### ISSUE TITLE
Lobe ML Kit Hardware Assembly Instructions

### ISSUE THREAD

**Issue by chagen24** [lobe/lobe-adafruit-kit#5]
The Lobe ML Kit comes with multiple pieces of hardware that need to be connected, but as far as I can tell, there are no assembly instructions. It would be helpful to have a quick assembly guide for anyone unfamiliar with connecting pieces of hardware together like this.

**Comment by bernardleung** [comment]
Agreed, I made wrong connection on the camera cable and the installation failed
